src/core/components/table/AttendanceTable/Composables/AttendanceSheets.js

```diff
@@ -78,6 +85,12 @@ export const useAttendanceSheets = (
     return attendanceSheets.value;
   });
 
+  const notLinkedSlotOptions = computed(() => {
+    if (!isSingleCourse.value) return [];
+
+    return course.value.slots.filter(s => attendanceSheets.value.every(as => !get(as, 'slots', []).includes(s._id)));
```

Comment:
Ici je pense que tu n'as pas besoin du get et tu peux ajouter des parentheses pour dire que le ! s'applique sur le resultat de ton `include` : !(as.slots.includes(s._id))

Reply:
Si je ne fais pas cette logique j'ai cette erreur qui sort en console :
TypeError: Cannot read properties of undefined (reading 'includes')

Reply:
Ah oui t'as raison ca pete pour une formation indiv qui a des FE non rattachées a des creneaux ? C'est ce que tu as testé ?